consensus/mint-client/src/bin/main.rs

```diff
@@ -186,8 +211,10 @@ fn main() {
         },
 
         Commands::HashMintTx { params } => {
+            // Fixme: we should panic if fog css is supplied I think, since this
+            // hash won't be deterministic then.
```

Comment:
I need to check how clap handles this, it may already give an error if an unexpected parameter is supplied?

Reply:
it looks like this already works as intended:

```
chris@06336bb308b1:/tmp/mobilenode$ ./target/docker/debug/mc-consensus-mint-client hash-mint-tx --fog-ingest-enclave-css
2022-10-13 23:23:09.041034155 UTC INFO mc-consensus-mint-client started: { "GIT_COMMIT": "c1cd7b2e-modified", "MOBILECOIN_GIT_COMMIT": "c1cd7b2e-modified", "PROFILE": "debug", "DEBUG": "true", "OPT_LEVEL": "0", "DEBUG_ASSERTIONS": "false", "TARGET_ARCH": "x86_64", "TARGET_OS": "linux", "TARGET_FEATURE": "adx,aes,avx,avx2,bmi1,bmi2,cmpxchg16b,ermsb,f16c,fma,fxsr,llvm14-builtins-abi,lzcnt,movbe,pclmulqdq,popcnt,rdrand,rdseed,sse,sse2,sse3,sse4.1,sse4.2,ssse3,xsave,xsavec,xsaveopt,xsaves", "RUSTFLAGS": "?", "SGX_MODE": "SW", "IAS_MODE": "DEV" }, mc.app: mc-consensus-mint-client, mc.module: mc_common::logger::loggers, mc.src: common/src/logger/loggers/mod.rs:249
error: Found argument '--fog-ingest-enclave-css' which wasn't expected, or isn't valid in this context

  If you tried to supply '--fog-ingest-enclave-css' as a value rather than a flag, use '-- --fog-ingest-enclave-css'

Usage: mc-consensus-mint-client hash-mint-tx [OPTIONS] --recipient <RECIPIENT> --token-id <TOKEN_ID> --amount <AMOUNT>
```

Reply:
```suggestion
```